src/components/input/input.scss

```diff
@@ -85,6 +85,13 @@
     block-size: auto;
   }
 
+  /**
+   * Removes ability to resize element
+   */
+  &.c-input--noResize {
```

Comment:
I believe this should be `c-input--no-resize` to conform to our class naming conventions?